libexec/bats-core/bats-format-tap-stream

```diff
@@ -20,15 +20,15 @@ fi
 
 update_screen_width() {
   screen_width="$(tput cols)"
-  count_column_left=$(( $screen_width - $count_column_width ))
+  count_column_left=$(( screen_width - count_column_width ))
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2004
Here and below - variables in math expression do not need `$`. See the link above for rationale.